packages/grafana-ui/src/components/uPlot/config/UPlotThresholds.ts

```diff
@@ -58,13 +58,13 @@ export function getThresholdsDrawHook(options: UPlotThresholdOptions) {
         color.setAlpha(0.7);
       }
 
-      const isVertical = u.scales.x ? u.scales.x.ori !== ScaleOrientation.Vertical : true;
+      const isHorizontal = u.scales.x!.ori === ScaleOrientation.Horizontal;
```

Comment:
in uPlot land, the chart orientation always follows the x axis.

for BarChart specifically the orientation we show in the UI/toggle follows the bars orientation. yes, it's inverted from the uPlot view of it. but since this code is for a uPlot wrapper, let's stick with the uPlot frame of reference.